src/screens/ProfileScreen.tsx

```diff
@@ -90,17 +93,17 @@ const ProfileScreen = () => {
                             borderColor="gray.200"
                             borderWidth={1}
                         >
-                            <Avatar.Badge bg="blue.500">
-                                <IconButton
-                                    size="xs"
-                                    _icon={{
-                                        as: MaterialIcons,
-                                        name: 'edit',
-                                        color: 'white'
-                                    }}
-                                    onPress={() => setEditProfile(true)}
-                                />
-                            </Avatar.Badge>
```

Comment:
I think its a good option to leave the edit button next to the profile image. We will upload a profile image through the `EditProfileMenu`